Changelog.md

```diff
@@ -7,11 +7,14 @@ This changelog should be updated as part of a PR if the work is worth noting (mo
 ## Next Release (Date TBD)
 
 #### New experimental features
-- [#615](https://github.com/timescale/timescaledb-toolkit/pull/614): Heatbeat aggregate
+- [#615](https://github.com/timescale/timescaledb-toolkit/pull/615): Heatbeat aggregate
```

Comment:
Hehe we caught the same thing!  Also "heatbeat" though, and also there are some rendering problems without blank lines.  I sent a pull request for these.